app/helpers/invite_helper.rb

```diff
@@ -0,0 +1,19 @@
+module InviteHelper
+  def invite_booth
+    if booth_params[:invited_users]
+      emails_array = booth_params[:invited_users].split(',')
+
+      emails_array.each do |email|
+        new_user = User.find_by(email: email)
+        if new_user.nil? || !new_user.confirmed?
+          User.invite!({ email: email }, current_user)
+        end
+        created_user = User.find_by(email: email)
+        if created_user && @booth.responsible_ids.exclude?(created_user.id)
+          BoothRequest.create(booth_id: @booth.id, user_id: created_user.id,
```

Comment:
Are you created a new booth request, just to add a booth responsible?

Reply:
@differentreality Whenever we add users to booth responsibles (not during invitation), a new booth request is created with same booth id but with different user ids. 
So I did the same for invited users also.

Reply:
@differentreality Is this a wrong approach? I have confirmed that adding booth responsible creates new booth request for those users.